website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -0,0 +1,169 @@
+---
+id: nevergrad_sweeper
+title: Nevergrad Sweeper plugin
+sidebar_label: Nevergrad Sweeper plugin
+---
+
+[Nevergrad](https://facebookresearch.github.io/nevergrad/) is a derivative-free optimization platform proposing a library of state-of-the art algorithms for hyperparameter search. This plugin provides a mechanism for Hydra applications to use [Nevergrad](https://facebookresearch.github.io/nevergrad/) algorithms for the optimization of experiments/applications parameters.
+
+Install with:
+
+```
+pip install hydra_nevergrad_sweeper
+```
+
+Once installed, override `hydra/sweeper` in your config:
+
+```yaml
+defaults:
+  - hydra/sweeper: nevergrad
+```
+
+## Example of training using Nevergrad hyperparameter search
+
+We include an example of how to use this plugin. The file [`example/dummy_training.py`](plugins/hydra_nevergrad_sweeper/example/dummy_training.py) implements an example of how to perform minimization of a (dummy) function including a mixture of continuous and discrete parameters. 
+
+
+This application has the following configuration:
+```yaml
+defaults:
+  - hydra/sweeper: nevergrad-sweeper
+
+hydra:
+  sweeper:
+    params:
+      # configuration of the optimizer
+      optim:
+        # name of the nevergrad optimizer to use
+        # OnePlusOne is good at low budget, but may converge early
+        optimizer: OnePlusOne
+        # total number of function evaluations to perform
+        budget: 100
+        # number of parallel workers for performing function evaluations
+        num_workers: 10
+      # default parametrization of the search space
+      parametrization:
+        # either one or the other
+        db:
+          - mnist
+          - cifar
+        # a log-distributed positive scalar, evolving by factors of 2 on average
+        lr:
+          init: 0.02
+          step: 2.0
+          log: true
+        # a custom nevergrad parameter evaluated at runtime (avoid using this whenever possible)
+        dropout: Scalar(lower=0.0, upper=1.0)
+        # an integer scalar going from 4 to 16
+        # init is set to the middle of the range
+        # step is set to a sixth of the range
+        batch_size:
+          lower: 4
+          upper: 16
+          integer: true
```

Comment:
commends seems out of sync with the parameters below.

Reply:
it's not, is it clearer now?
```yaml
# an integer scalar going from 4 to 16
# init and step parameters could also be provided,
# by default init is set to the middle of the range
# and step is set to a sixth of the range
```

Reply:
Yup. this is much better.